package androidx.compose.material.icons.filled;

import bj.n;
import g1.m0;
import g1.t;
import j0.a;
import k0.c;
import k1.e;
import k1.f;
import k1.h0;
import kotlin.jvm.internal.l;

/* JADX INFO: compiled from: r8-map-id-cf66f36031e302b9a7b5a76da57d208ae1b6ba4a38bba3bee1d4ed6983c54df3 */
/* JADX INFO: loaded from: classes.dex */
public final class DisabledByDefaultKt {
    private static f _disabledByDefault;

    public static final f getDisabledByDefault(a aVar) {
        l.f("<this>", aVar);
        f fVar = _disabledByDefault;
        if (fVar != null) {
            return fVar;
        }
        e eVar = new e("Filled.DisabledByDefault", 24.0f, 24.0f, 24.0f, 24.0f, 0L, 0, false, 224);
        int i10 = h0.f9067a;
        m0 m0Var = new m0(t.f6905b);
        n nVarG = k0.f.g(3.0f, 3.0f, 18.0f, 18.0f, 3.0f);
        c.c(nVarG, 3.0f, 17.0f, 15.59f);
        nVarG.l(15.59f, 17.0f);
        nVarG.l(12.0f, 13.41f);
        nVarG.l(8.41f, 17.0f);
        nVarG.l(7.0f, 15.59f);
        nVarG.l(10.59f, 12.0f);
        nVarG.l(7.0f, 8.41f);
        nVarG.l(8.41f, 7.0f);
        nVarG.l(12.0f, 10.59f);
        nVarG.l(15.59f, 7.0f);
        nVarG.l(17.0f, 8.41f);
        c.p(nVarG, 13.41f, 12.0f, 17.0f, 15.59f);
        e.a(eVar, nVarG.f3166a, 0, m0Var);
        f fVarB = eVar.b();
        _disabledByDefault = fVarB;
        return fVarB;
    }
}
